torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -18,23 +19,16 @@ XLATensor& GetXlaTensor(const at::Tensor& tensor) {
   return impl->tensor();
 }
 
-std::vector<at::Tensor> XlaCreateTensorList(const at::TensorList& tensors,
-                                            const std::vector<bool>* writeable) {
+std::vector<at::Tensor> XlaCreateTensorList(
+    const at::TensorList& tensors, const std::vector<bool>* writeable) {
   std::vector<XLATensor> xla_tensors;
   for (auto& tensor : tensors) {
-    xla_tensors.push_back(GetXlaTensor(tensor));
+    xla_tensors.push_back(tensor.defined() ? GetXlaTensor(tensor)
```

Comment:
Let's not push the junk tensor into the XLA tensor core, and add handling there.
This can be handled here.